api/src/test/resources/test-application.properties

```diff
@@ -72,6 +72,7 @@ saml.authorities.attribute.names=test_name
 saml.authn.request.binding=
 saml.sign.key=
 saml.user.attributes=email=test_email,user=test_user
+saml.user.blocked.attribute=false
```

Comment:
Do we need to specify this value here if we still use `ReflectionTestUtils.setField`?

Reply:
Done